### PR TITLE
Improvement/create class with database connection

### LINKED ISSUE #7: Create a class with database connection
To create a class for database connection with propper functions and with procedure usage. 
The best way is to use singleton design pattern for creating an instance once and comming back to it with every call

### PR DESCRIPTION
**Pull request by Cialkowski**
Created a class with working database connection 
image

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@Cialkowski
```
Comment (Borzygniew) [inline CODEOWNERS]: why I see this here? you didnt change codeowners. If you want do this here why u opened pr with codeowners before?

File: .gitignore [hunk 2]
```diff
@@ -1,32 +1,398 @@
...
 
-# Compiled Object files
-*.slo
-*.lo
-*.o
-*.obj
+# User-specific files
+*.rsuser
+*.suo
+*.user
```
Comment (Borzygniew) [inline .gitignore]: same here

**Review by Borzygniew — CHANGES_REQUESTED** [review]
delete previous pull requests or remove codeowners, gitignore etc files

File: CODEOWNERS (hunk 1 above)
Comment (Cialkowski) [reply CODEOWNERS]: done

File: .gitignore (hunk 2 above)
Comment (Cialkowski) [reply .gitignore]: done

**Comment by Cialkowski** [comment]
> delete previous pull requests or remove codeowners, gitignore etc files

Done

File: UberBarber/UberBarber/DbConnection.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    case 1045:
+                        MessageBox.Show("Invalid username/password, please try again");
+                        break;
+                }
+                return false;
+                
+            }
+        }
+
+        private bool close_connection()
```
Comment (Borzygniew) [inline UberBarber/UberBarber/DbConnection.cs:68]: docstring?

File: UberBarber/UberBarber/DbConnection.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            _database = "serwer165956_projektstudia";
+            _uid = "serwer165956_projektstudia";
+            _password = "Abc1234!";
+            string connection_string = "SERVER=" + _server + ";" + "DATABASE=" +
+                                       _database + ";" + "UID=" + _uid + ";" + "PASSWORD=" + _password + ";";
+
+            _connection = new MySqlConnection(connection_string);
+        }
+
+        private bool open_connection()
```
Comment (Borzygniew) [inline UberBarber/UberBarber/DbConnection.cs:41]: docstring?

File: UberBarber/UberBarber/DbConnection.cs (hunk 3 above)
Comment (Cialkowski) [reply UberBarber/UberBarber/DbConnection.cs:68]: done

File: UberBarber/UberBarber/DbConnection.cs (hunk 4 above)
Comment (Cialkowski) [reply UberBarber/UberBarber/DbConnection.cs:41]: done

**Review by Borzygniew — APPROVED** [review]
Zrzut ekranu 2023-01-11 213744

**Review by goflyy — APPROVED** [review]

**Pull request merged**